src/metabase/api/card.clj

```diff
@@ -711,7 +711,7 @@
    format_rows   [:maybe :boolean]
    export-format (into [:enum] api.dataset/export-formats)}
   (qp.card/process-query-for-card
-   card-id export-format
+   card-id (keyword export-format)
```

Comment:
Not really related but in my quest to wrap more stuff in `mu/defn` to debug test failures I made a schema for `export- format` and now most things expect a keyword so we have to actually make sure we convert it